apps/subscription.go

```diff
@@ -139,24 +140,22 @@ func (e Event) validate(appendTo error) error {
 	// Team scoped, require TeamID, no ChannelID
 	case SubjectUserJoinedTeam,
 		SubjectUserLeftTeam,
-		SubjectBotJoinedChannel,
-		SubjectBotLeftChannel,
 		SubjectChannelCreated:
 		if e.TeamID == "" {
-			appendTo = multierror.Append(appendTo, utils.NewInvalidError("%s is scoped to a team; teamID must not be empty", e.Subject))
+			appendTo = multierror.Append(appendTo, utils.NewInvalidError("%s is scoped to a team; team_id must not be empty", e.Subject))
```

Comment:
is this an error? I see the element is called `TeamID`, not `team_id`? In line 140 is also called that way.

Reply:
The JSON is lowercase.